### PR TITLE
Adds <blockquote> and <cite> to  Lucky::BaseTags

### LINKED ISSUE #1316: Missing macros for some html tags
It seems `<blockquote>` and `<cite>` don't have their respective macros. Should we give them some magic love ?

### PR DESCRIPTION
**Pull request by rmarronnier**
Fixes #1316 
Adds `<blockquote>` and `<cite>` html tags to `Lucky::BaseTags`.

## Checklist
* [X] - An issue already exists detailing the issue/or feature request that this PR fixes
* [X] - All specs are formatted with `crystal tool format spec src`
* [ ] - Inline documentation has been added and/or updated
* [X] - Lucky builds on docker with `./script/setup`
* [X] - All builds and specs pass on docker with `./script/test`

### CODE REVIEW

**Review by stephendolan — APPROVED** [review]
This looks good to me. Thank you for keeping the list in alphabetical order 👍 

We'll wait for one more approval and then get it merged!

**Review by matthewmcgarvey — APPROVED** [review]

**Pull request merged**
